src/kernel.cu

```diff
@@ -56,6 +56,10 @@ __global__ void warp(ImagePtrCUDA<T> srcPtr, ImagePtrCUDA<T> dst, ImagePtrCUDA<f
         T gray  = (T)CV_DESCALE(b * BY15 + g * GY15 + r * RY15, GRAY_SHIFT);
         float gray_scale   = ((float)gray - IMG_MEAN_GRAY) * IMG_SCALE_GRAY;
         *dst_gray.ptr(dst_b, y, x, 0) = gray_scale;
+        // bgr norm
```

Comment:
Is this a bug in previous version ?

Reply:
no, previous OCRnet uses gray as input, but CLIP4STR needs the RGB norm data as input.